charts/cytomine/templates/core/core.yaml

```diff
@@ -28,30 +28,38 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       volumes:
-      - name: singularity-images
-        persistentVolumeClaim:
-          claimName: {{ template "cytomine.name" . }}-singularity-claim
       - name: core-config
         secret:
           secretName: {{ include "cytomine.name" . }}-core-config
+      - name: {{ include "cytomine.name" . }}-core-localtime
+        hostPath:
+          path: /etc/localtime
```

Comment:
Do we really need this? Can we bring the corresponding `tzfile` in through a `ConfigMap` if so?

Reply:
Resolved.